src/sidebar/components/focused-mode-header.js

```diff
@@ -4,42 +4,59 @@ const { createElement } = require('preact');
 
 const useStore = require('../store/use-store');
 
+const SvgIcon = require('./svg-icon');
+
 /**
  * Render a control to interact with any focused "mode" in the sidebar.
  * Currently only a user-focus mode is supported but this could be broadened
  * and abstracted if needed. Allow user to toggle in and out of the focus "mode."
  */
 function FocusedModeHeader() {
-  const store = useStore(store => ({
-    actions: {
-      setFocusModeFocused: store.setFocusModeFocused,
-    },
-    selectors: {
-      focusModeFocused: store.focusModeFocused,
-      focusModeUserPrettyName: store.focusModeUserPrettyName,
-    },
+  const actions = useStore(store => ({
+    setFocusModeFocused: store.setFocusModeFocused,
+  }));
+  const selectors = useStore(store => ({
+    focusModeFocused: store.focusModeFocused(),
+    focusModeHasUser: store.focusModeHasUser(),
+    focusModeUserPrettyName: store.focusModeUserPrettyName(),
   }));
 
+  // Nothing to do here for now if we're not focused on a user
+  if (!selectors.focusModeHasUser) {
+    return null;
+  }
+
   const toggleFocusedMode = () => {
-    store.actions.setFocusModeFocused(!store.selectors.focusModeFocused());
+    actions.setFocusModeFocused(!selectors.focusModeFocused);
   };
 
-  const buttonText = () => {
-    if (store.selectors.focusModeFocused()) {
-      return `Annotations by ${store.selectors.focusModeUserPrettyName()}`;
+  const filterStatus = (
+    <div className="focused-mode-header__filter-status">
+      {selectors.focusModeFocused ? (
+        <span>
+          Annotations by <strong>{selectors.focusModeUserPrettyName}</strong>{' '}
+          only
+        </span>
+      ) : (
+        <span>Everybody&rsquo;s annotations</span>
+      )}
+    </div>
+  );
+
+  const buttonText = (() => {
+    if (selectors.focusModeFocused) {
+      return 'Show all';
     } else {
-      return 'All annotations';
+      return `Only ${selectors.focusModeUserPrettyName}`;
     }
-  };
+  })();
 
   return (
-    <div className="focused-mode-header">
-      <button
-        onClick={toggleFocusedMode}
-        className="primary-action-btn primary-action-btn--short"
-        title={`Toggle to show annotations only by ${store.selectors.focusModeUserPrettyName()}`}
-      >
-        {buttonText()}
+    <div className="focused-mode-header sheet">
+      {filterStatus}
+      <button onClick={toggleFocusedMode} className="focused-mode-header__btn">
```

Comment:
Looks really reasonable, nothing stands out to me. My only suggestion if we could start baking in some of the ARIA labels. Since this is a toggle button we could add aria-pressed and set that to true when its filtering to a user, and false when its not.

Reply:
Specific to `aria-pressed`, I'm not sure it makes sense: the button is never really active/pressed as it always represents the mode that _isn't_ applied. It's not just the state that toggles; it's the button _itself_.

Reply:
... or, put another way, `aria-pressed` would always be `false`

Reply:
This makes sense, and because the text changes, that would be sufficient to a reader to convey the state of the button. This is not a true toggle button